### PR TITLE
[STY] apply black 24.1.0

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none

Changes proposed in this pull request:

- upgrade black version in precommit hooks and apply it

### CODE REVIEW

File: nilearn/datasets/func.py [hunk 1]
```diff
@@ -2371,12 +2402,22 @@ def patch_openneuro_dataset(file_list):
...
+        ),  # noqa 501
         "_bold_confounds": "_desc-confounds_regressors",
-        "_bold_space-MNI152NLin2009cAsym_brainmask": "_space-MNI152NLin2009cAsym_desc-brain_mask",  # noqa 501
-        "_bold_space-MNI152NLin2009cAsym_preproc": "_space-MNI152NLin2009cAsym_desc-preproc_bold",  # noqa 501
+        "_bold_space-MNI152NLin2009cAsym_brainmask": (
+            "_space-MNI152NLin2009cAsym_desc-brain_mask"
+        ),  # noqa 501
+        "_bold_space-MNI152NLin2009cAsym_preproc": (
+            "_space-MNI152NLin2009cAsym_desc-preproc_bold"
+        ),  # noqa 501
```
Comment (ymzayek) [inline nilearn/datasets/func.py:2420]: we should now be able to remove all the ignore E501 errors from all of these right?

Comment (Remi-Gau) [reply nilearn/datasets/func.py:2420]: yup

**Comment by Remi-Gau** [comment]
FYI, I would probably wait for the official release of the new black before applying this.

**Comment by Remi-Gau** [comment]
OK we are getting a failure on main because the new version of black is out.

Will update this PR to fix it.

**Review by bthirion — APPROVED** [review]
This fixes some ugly lines ;-)

**Comment by Remi-Gau** [comment]
> This fixes some ugly lines ;-)

I know: maybe in 10 years we will have auto formatting that's also pleasant to the eye.

**Pull request merged**